filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilder.scala

```diff
@@ -39,28 +40,31 @@ object BlobPathBuilder {
     *
     * If the configured container and storage account do not match, the string is considered unparsable
     */
-  def validateBlobPath(string: String, container: BlobContainerName, endpoint: EndpointURL): BlobPathValidation = {
+  def validateBlobPath(string: String): BlobPathValidation = {
     val blobValidation = for {
       testUri <- parseURI(string)
-      endpointUri <- parseURI(endpoint.value)
+      testEndpoint = EndpointURL(testUri.getScheme + "://" + testUri.getHost())
       testStorageAccount <- parseStorageAccount(testUri)
-      endpointStorageAccount <- parseStorageAccount(endpointUri)
-      hasContainer = testUri.getPath.split("/").find(_.nonEmpty).contains(container.value)
-      hasEndpoint = testStorageAccount.equals(endpointStorageAccount)
-      blobPathValidation = (hasContainer && hasEndpoint) match {
-        case true => ValidBlobPath(testUri.getPath.replaceFirst("/" + container, ""))
-        case false => UnparsableBlobPath(new MalformedURLException(invalidBlobPathMessage(container, endpoint)))
+      testContainer = testUri.getPath.split("/").find(_.nonEmpty)
+      isBlobHost = testUri.getHost().contains(".blob.core.windows.net") && testUri.getScheme().contains("https")
```

Comment:
This should be totally safe for our purposes in the short term, but burying this extremely important string here makes me nervous. [In theory we could end up with other URLs](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-overview?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json#azure-dns-zone-endpoints-preview). I suggest moving it to a constant at the top of the class, and making it a list that includes `z[00-50].blob.storage.azure.net` if you are in a paranoid mood (doesn't feel totally necessary to me given that that feature is in preview and we don't use it in Terra).